feinstaub/feinstaub/settings/base.py

```diff
@@ -56,10 +58,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'feinstaub',
-        'USER': 'postgres',
-        'HOST': 'db',
-        'PORT': 5432,
+        'NAME': os.getenv("POSTGRESQL_DATABASE",'feinstaub'),
+        'USER': os.getenv("POSTGRESQL_USERNAME", 'postgres'),
+        'HOST': os.getenv("POSTGRESQL_HOST", 'db'),
+        'PASSWORD': os.getenv("POSTGRESQL_PASSWORD", ""),
+        'PORT': os.getenv("POSTGRESQL_PORT", 5432),
```

Comment:
Let's do this config with `DATABASE_URL` instead. i.e https://github.com/kennethreitz/dj-database-url